src/Content/Version.php

```diff
@@ -98,6 +114,27 @@ public function id(): VersionId
 		return $this->id;
 	}
 
+	/**
+	 * Converts a "user-facing" language code to a `Language` object
+	 * to be used in storage methods
+	 */
+	protected function language(
+		string|null $languageCode = null,
```

Comment:
I really like that idea. I stumbled across it a couple times later where it would have been nice to simply pass the Language object. But I did not implement it because it would not have fitted in any other PR. I think it's a good idea to already take care of it in this one.

Reply:
✅